k8s/trillian/schema.yaml

```diff
@@ -0,0 +1,133 @@
+application_api_version: v1beta1
+properties:
+  NAME:
+    type: string
+    x-google-marketplace:
+      type: NAME
+  NAMESPACE:
+    type: string
+    x-google-marketplace:
+      type: NAMESPACE
+  # TODO(RJPercival): One image must be the "primary" image and have the same
+  # name as the product (i.e. "trillian"). The remaining images must have names
+  # nested under that, e.g. "trillian/signer".
+  IMAGE_TRILLIAN_LOGSERVER:
+    type: string
+    default: $REGISTRY/log_server:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  IMAGE_TRILLIAN_LOGSIGNER:
+    type: string
+    default: $REGISTRY/log_signer:$TAG
+    x-google-marketplace:
+      type: IMAGE
+  TRILLIAN_LOGSERVER_REPLICAS:
+    type: integer
+    title: Trillian replicas
+    description: The number of Trillian log servers to run
+    default: 4
+    minimum: 1
+  TRILLIAN_LOGSIGNER_REPLICAS:
+    type: integer
+    title: Trillian replicas
+    description: The number of Trillian log signers to run
+    default: 2
+    minimum: 1
+  IMAGE_ETCD_OPERATOR:
+    type: string
+    default: quay.io/coreos/etcd-operator:v0.9.1
+    x-google-marketplace:
+      type: IMAGE
+  ETCD_CLUSTER_SIZE:
```

Comment:
Should there be a recommendation to use odd numbers for the cluster size? If yes, you could use the description field for a recommendation displayed only after hovering a question mark icon.

Reply:
Yes, done.